src/main/java/Bundle.properties

```diff
@@ -179,7 +179,7 @@ authenticationProvidersAvailable.tip={0}There are no active authentication provi
 login.System=Login System
 login.forgot.text=Forgot your password?
 login.institution=Institution Log In
-login.invaliduserpassword=The username and/or password you entered is invalid. Need assistance accessing your account?
+login.invaliduserpassword=The username, email address, or password you entered is invalid. Need assistance accessing your account?
 login.error=Error validating the username and/or password. Please try again. If the problem persists, contact an administrator.
```

Comment:
I'm not quite sure how to exercise this key. Something with password upgrade, I think.